contrib/generic_proxy/filters/network/source/proxy.cc

```diff
@@ -303,12 +305,13 @@ void ActiveStream::completeRequest() {
   }
 }
 
-Envoy::Network::FilterStatus Filter::onData(Envoy::Buffer::Instance& data, bool) {
+Envoy::Network::FilterStatus Filter::onData(Envoy::Buffer::Instance& data, bool end_stream) {
   if (downstream_connection_closed_) {
     return Envoy::Network::FilterStatus::StopIteration;
   }
 
-  request_decoder_->decode(data);
+  // Basically this end_stream should always be false because we don't support half-close.
```

Comment:
This can be an assertion

Reply:
If the end_stream effect our following logic, then I think an assertion would be more reasonable. But it doesn't here. May if it ok I prefer just leave this like that. 🤣